clang/test/CodeGen/attr-target-version.c

```diff
@@ -673,7 +673,7 @@ int hoo(void) {
 //
 //
 // CHECK: Function Attrs: noinline nounwind optnone
-// CHECK-LABEL: define {{[^@]+}}@fmv_inline._MlseMrdm
+// CHECK-LABEL: define {{[^@]+}}@fmv_inline._MlseMrdma
```

Comment:
It seems potentially problematic that this leaks into the mangling. Maybe these ought to be canonicalized in `AppendTargetVersionMangling` and `AppendTargetClonesMangling` with a `resolveExtAlias` call.

Reply:
What does GCC do?

Reply:
I raised this with the GCC team, I don't have an answer yet but I believe GCC only accepts "rdma" on the attribute (so I suppose that makes its way to the mangled name?). I remember you raised this matter again in a previous review, when we changed priorities and the mangling was affected. That is a good point but I think first we need to agree whether those functions are part of the ABI or not. Right now I don't see why they need to be. We could even make then have internal linkage, no? If I am mistaken a counter example would help, thanks!

Reply:
Right, I didn't know that the resolver contains references to every target-version declaration that is visible in its translation unit even if the definition actually resides in another translation unit. That would be a problem if the definition has a mangled name that doesn't match because of the alias. Thanks for clarifying!